Formula/uwsgi.rb

```diff
@@ -26,16 +26,9 @@ class Uwsgi < Formula
   depends_on "pkg-config" => :build
   depends_on "openssl@1.1"
   depends_on "pcre"
-  depends_on "python@2"
+  depends_on "python"
   depends_on "yajl"
 
-  # "no such file or directory: '... libpython2.7.a'"
-  # Reported 23 Jun 2016: https://github.com/unbit/uwsgi/issues/1299
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/726bff4/uwsgi/libpython-tbd-xcode-sdk.diff"
-    sha256 "d71c879774b32424b5a9051ff47d3ae6e005412e9214675d806857ec906f9336"
-  end
-
   def install
     # Fix file not found errors for /usr/lib/system/libsystem_symptoms.dylib and
```

Comment:
Since homebrew doesn't support Sierra and El Capitan, do we still need this?

Reply:
For older versions: we don't actively remove things that make them work. We don't build bottles, and we don't offer support, but we will keep them working if it has no cost for us, basically.

Reply:
Got it, thanks for the explanation!